BurnOutSharp/ProtectionType/Macrovision.cs

```diff
@@ -266,6 +268,224 @@ private string CheckSectionForProtection(string file, bool includeDebug, List<st
             return MatchUtil.GetFirstMatch(file, sectionRaw, matchers, includeDebug);
         }
 
+        private List<string> FindProductFromVersion(List<string> resultsList)
+        {
+            // If we have an invalid result list
+            if (resultsList == null || resultsList.Count == 0)
+                return resultsList;
+
+            List<string> VerifiedCDS300Versions = new List<string>()
```

Comment:
Local variables shouldn't start with an upper-case letter. This goes for all of your lists here.